src/server.tsx

```diff
@@ -376,6 +380,34 @@ app.post('/epic/compare-variant-decision', async (req: express.Request, res: exp
     res.send('thanks');
 });
 
+app.get(
+    '/amp/experiments_data',
+    cors({
+        origin: [
+            'https://amp-theguardian-com.cdn.ampproject.org',
+            'https://amp.theguardian.com',
+            'http://localhost:3030',
+            'https://amp.code.dev-theguardian.com',
+        ],
+        credentials: true,
+        allowedHeaders: ['x-gu-geoip-country-code'],
+    }),
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
I think for endpoints we should drop the call to `next` (there shouldn't be anything after in the middleware stack anyways)

Reply:
I does look like we do this everywhere though so 🤷‍♂️. Perhaps we just leave this 😂. @tomrf1 do you know why all our endpoints are like this?

Reply:
I think we should keep these in for the time being, if only in the name of consistency across the server file. We could look at refactoring in a separate PR though :)